app/services/manifest_builder.rb

```diff
@@ -635,6 +635,22 @@ def geotiff_child
         end
     end
 
+    def cropped_geotiff_child
```

Comment:
Question: Would it make sense to move a bunch of this navigation logic to the wayfinder? So we can do like Wayfinder.for(resource).try(:cropped_geotiff_child)`?

Reply:
Good idea. Done.